.github/actions/build-matrix/action.yml

```diff
@@ -132,9 +132,8 @@ runs:
         fi
 
         MATRIX=$(cat matrix.json)
+        echo 'matrix<<EOF' >> $GITHUB_OUTPUT
+        cat matrix.json >> $GITHUB_OUTPUT
+        echo 'EOF' >> $GITHUB_OUTPUT
```

Comment:
Note sure why this part is not working
@mshima any clue?

Reply:
matrix.json file doesn't contains a trailing EOL.